run-docker-sample.sh

```diff
@@ -18,26 +18,51 @@
 #
 # ------------------------------------------------------------------------
 
+service_provider_name=tomcat-saml-agent-sample
```

Comment:
Add a description on what this script does

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/13/commits/d14f65689a5380fe93f51e02ff4e1c77e255c505